app/build.gradle

```diff
@@ -127,6 +127,9 @@ dependencies {
     implementation 'com.github.bumptech.glide:glide:4.8.0'
     kapt 'com.github.bumptech.glide:compiler:4.8.0'
 
+    // Circle Image View
+    implementation 'de.hdodenhof:circleimageview:3.0.1'
```

Comment:
I think this dependency should be added in another PR as you also used this in your previous PR.

Reply:
to satisfy the need of this task, i don't think removing it is justified. I will solve the merge conflict when it arises.